examples/public.php

```diff
@@ -0,0 +1,61 @@
+<?php
+// Example - get all public wall links - useful for repost links pulbics
+session_start();
+
+require '../vendor/autoload.php';
+
+use \BW\Vkontakte as Vk;
+
+$vk = new Vk([
+    'client_id' => 'XXXXXX',
+    'client_secret' => 'XXXXXX',
+    'redirect_uri' => 'http://localhost/vkontakte-php-sdk/examples/',
+]);
+$public = '';
+
+if (isset($_GET['code'])) {
+    $vk->authenticate();
+    $_SESSION['access_token'] = $vk->getAccessToken();
+    header('Location: ' . $_SERVER['REQUEST_SCHEME'] . '://' . $_SERVER['HTTP_HOST'] . $_SERVER['PHP_SELF']);
+    exit;
+} else {
+    $vk->setAccessToken($_SESSION['access_token']);
```

Comment:
Let's use:
``` php
$accessToken = isset($_SESSION['access_token']) ? $_SESSION['access_token'] : null;
$vk->setAccessToken($accessToken);
```

Otherwise, we get a warning message